tools/topology/topology2/cavs/platform/intel/hdmi-default.conf

```diff
@@ -0,0 +1,31 @@
+# Default HDMI variable definitions
+Define {
+	HDMI1_ID			5
+	HDMI2_ID			6
+	HDMI3_ID			7
+	HDMI4_ID			8
+	HDMI1_HOST_PIPELINE_ID		20
+	HDMI1_DAI_PIPELINE_ID		21
+	HDMI2_HOST_PIPELINE_ID		22
+	HDMI2_DAI_PIPELINE_ID		23
+	HDMI3_HOST_PIPELINE_ID		24
+	HDMI3_DAI_PIPELINE_ID		25
+	HDMI4_HOST_PIPELINE_ID		26
+	HDMI4_DAI_PIPELINE_ID		27
+	HDMI1_PCM_CAPS			"Passthrough Playback 20"
```

Comment:
Is there anyway we can use a macro for the ```20``` here and so on as it seems we are using this for string matching. Btw where doe the 20 come from ?

Reply:
We need to define this ```20``` as a macro somewhere so that the other HDMI1 definitions can reuse it.

Reply:
> We need to define this `20` as a macro somewhere so that the other HDMI1 definitions can reuse it.

@lgirdwood It seems we have no way to define a macro in .conf file. @ranj063 Any idea?

Reply:
ok, after MTL we can circle back on these areas and improve them for later on